### DISCUSSION TITLE
Reset to tare the scale - DONE

### DISCUSSION THREAD

**Discussion by reganissu** [pisto/blastic-scale#5] [Ideas]
Get the scale tared to zero on reset botton push

**Comment by pisto** [comment]
We need an extra menu for configuration/calibration etc. Sure you want to do an hard reset for a tare operation? It would stand out compared to the other management operations.

**Reply by reganissu** [reply]
To be true I don't mind tare operation to stand out comparing to all the other operations.  That's because the device in its essence is a scale. Reasoning:
1)  In my working experience the tare button have been by far the most continuously used 
2) In[ other scale libraries](https://github.com/bogde/HX711) that outstanding operation existed in the same way and tested beautifully
3) in functional (specialized) object design I think this operational "asymmetry" can be understood, compared to more systemic enviroment
